extensions/Article/Entity/Article.xml

```diff
@@ -37,13 +37,6 @@
                description="COM_CONTENT_FIELD_ARTICLETEXT_DESC"
         />
 
-        <field name="author"
-               type="text"
-               label="COM_CONTENT_FIELD_CREATED_BY_ALIAS_LABEL"
-               description="COM_CONTENT_FIELD_CREATED_BY_ALIAS_DESC"
-               default=""
-        />
-
```

Comment:
It is not needed as it is added from Users automatically. If you do php installation/demo.php and open http://joomla.box/j4/?id=1&entity=article, then you will see the author field.